ibm/resource_ibm_dns_reverse_record.go

```diff
@@ -0,0 +1,147 @@
+package ibm
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/softlayer/softlayer-go/datatypes"
+	"github.com/softlayer/softlayer-go/services"
+	"github.com/softlayer/softlayer-go/sl"
+)
+
+func resourceIBMDNSREVERSERecord() *schema.Resource {
+	return &schema.Resource{
+		Exists:   resourceIBMDNSREVERSERecordExists,
+		Create:   resourceIBMDNSREVERSERecordCreate,
+		Read:     resourceIBMDNSREVERSERecordRead,
+		Update:   resourceIBMDNSREVERSERecordUpdate,
+		Delete:   resourceIBMDNSREVERSERecordDelete,
+		Importer: &schema.ResourceImporter{},
+		Schema: map[string]*schema.Schema{
+			"dns_ipaddress": {
```

Comment:
rename dns_ipaddress to ipaddress  similarly other attributes dns_hostname to hostname , dns_ttl to ttl